### PR TITLE
[#19183] fix: scan qr design issues

### LINKED ISSUE #19183: Shell - Scan QR code screen issues
# Design Feedback 2.27

## [Shell] Scan QR code screen
Figma frame of the Review [here](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj/Release-Reviews?type=design&node-id=3%3A68309&mode=design&t=OeK3GHXzoVCNnZog-1)

- **Top Nav button - Left** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727573274)] should be ❌ not ⬅️ back arrow
- **Top Nav button - Right** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727573985)] is missing
- **Flash camera button** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727571527)] has the wrong style
- **Missing copy** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727570622)] under the scanning area
- **Overlay color** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#726033170)] is wrong

Screenshot 2024-03-11 alle 16 56 44

### Reproduction

1. Open the Scan QR code icon at the top right of the screen

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS
- Devices in the screenshots: iPhone 15 Pro Max

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by mohsen-ghafouri**
fixes #19183

### Summary

- Top Nav button - Left [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727573274)] should be ❌ not ⬅️ back arrow
- Top Nav button - Right [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727573985)] is missing
- Flash camera button [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727571527)] has the wrong style
- Missing copy [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#727570622)] under the scanning area
- Overlay color [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=111:82257&mode=design#726033170)] is wrong

#### Areas that maybe impacted

- Shell] Scan QR code screen

### Steps to test

- Open the Scan QR code icon at the top right of the screen

### Before and after screenshots

status: ready

### CODE REVIEW

File: src/status_im/common/scan_qr_code/view.cljs [hunk 1]
```diff
@@ -31,7 +31,17 @@
...
+     :i/close]
+    [quo/button
+     {:icon-only?          true
+      :type                :grey
+      :background          :blur
+      :size                32
+      :accessibility-label :show-qr-button
+      :on-press            (fn []
+                             (rf/dispatch [:navigate-back])
+                             (rf/dispatch [:open-modal :share-shell]))}
```
Comment (ibrkhalil) [inline src/status_im/common/scan_qr_code/view.cljs]: Can we rename this screen to be consistent with our naming convention for screens?
`:screen/share-shell` ?

**Review by ibrkhalil — APPROVED** [review]

File: src/status_im/common/scan_qr_code/view.cljs (hunk 1 above)
Comment (mohsen-ghafouri) [reply src/status_im/common/scan_qr_code/view.cljs]: done

**Comment by mohsen-ghafouri** [comment]
@status-im/mobile-qa could you please check test results?

**Review by jo-mut — APPROVED** [review]

**Comment by pavloburykh** [comment]
> @status-im/mobile-qa could you please check test results?

@mohsen-ghafouri thank you for the PR! E2E failures are not PR related. Ready for design review. cc @Francesca-G

**Review by Francesca-G — COMMENTED** [review]
All issues look fixed besides the overlay:

Screenshot 2024-04-22 alle 14 36 51

>
as you can see from the screenshot on the right the camera lines under the overlay are still clearly visible

**Comment by mohsen-ghafouri** [comment]
Thank you @Francesca-G, I fixed the blur color but for camera frame issue we have another open issue https://github.com/status-im/status-mobile/issues/17787

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**